src/compiler/function/ArrowFunction.ts

```diff
@@ -3,15 +3,30 @@ import { AsyncableNode, BodiedNode, JSDocableNode, ModifierableNode, SignaturedD
 import { Node } from "../common";
```

Comment:
Can you revert the changes to `ArrowFunction` as part of this pull request (`ArrowFunctionPrinter`, `ArrowFunctionStructure`, etc)? We can add it in a separate pull request.